SPIE2016-9913-16/9913-16.tex

```diff
@@ -0,0 +1,303 @@
+
+\section{Synergies}
+
+The Astropy and LSST software projects are both attempting to write astronomical software in Python that will benefit the community.
+In LSST's case we are constrained by the requirements of MREFC funding to deliver a data management system on schedule and on budget and we can justify involvement with Astropy if it allows us to deliver the functionality without impacting schedule, or if opening up the LSST algorithms to the Astropy community encourages early adoption (through ease of installation) which can lead to more extensive testing.
+Additionally, the community will be encouraged to try out the LSST software sooner if it integrates well with Astropy and this will significantly enhance usage of the Level 3 processing system during LSST operations.
+In this section we look at the areas where collaboration between LSST and Astropy can enhance both projects.
+
+\subsection{Generalized Word Coordinate Systems}
+
+JWST and LSST have requirements for accurate world coordinate conversions that cannot be represented in the current FITS-WCS framework\cite{2013ASPC..475...49H,2002A&A...395.1077C,2015A&C....12..133T} and none of the current extensions handling focal plane distortions are suitably flexible\cite{2012SPIE.8451E..1MS,2012tpv}.
+This lack of flexibility in FITS-WCS has been a long standing issue and the Starlink AST library\cite{Berry201633} provides one solution to this problem that has been in use for nearly 20 years.
+For JWST a pure-Python equivalent, GWCS, is in development\cite{2016_P028_adassxxv} and is not yet part of Astropy but builds on the \texttt{astropy.modeling} framework.
+LSST expects to require world coordinate transformations to be performed in the C++ layer for performance reasons, and this may preclude direct adoption of GWCS by LSST, given the currently existing Python-based implementation.
+It would be beneficial to both parties if the future LSST WCS implementation and GWCS could interoperate easily.
+We are therefore investigating a text-based serialization of WCS that could be usable as an interchange format.
+AST currently supports a number of serialization schemes including multiple dialects of FITS headers, STC-S\cite{2007ivoa.spec.1030R} for region handling\cite{2010ASPC..434..213B} and an internal text-based format.
+GWCS can serialize to FITS-WCS and also into YAML using the Advanced Scientific Data Format (ASDF)\cite{2015A&C....12..240G}.
+We have decided to investigate a common format that is based on YAML and the proposed new variant of STC, version 2.0.
+STC-2\cite{2015ivoaSTC2} is an evolution of STC that includes support for chainable mappings and seems like it could be a suitable starting point for this investigation.
+Both the Astropy and IVOA communities would benefit if such a standard could be established and it could be a significant factor in bringing together the competing distortion models currently in use in the FITS community.
```

Comment:
“AST” stands for “ASTrometry Library”. The name arose when it was thought that knowledge of “astrometry” (i.e. celestial coordinate systems) would form the bulk of the library. In fact, it turns out that astrometry forms only a minor component, but the name AST has stuck.